media_driver/agnostic/gen11/codec/hal/codechal_encode_csc_ds_g11.cpp

```diff
@@ -29,7 +29,7 @@
 #include "codechal_encode_csc_ds_g11.h"
 #include "codechal_kernel_header_g11.h"
 #include "codeckrnheader.h"
-#if defined(ENABLE_KERNELS)
+#ifndef _FULL_OPEN_SOURCE
```

Comment:
Driver has a build path to disable all kernels, including free kernels. This change will simply break this build in case when repo will be cleaned out from all binary sources like here: https://github.com/intel/media-driver/wiki/Media-Driver-Shaders-(EU-Kernels)#disable-shaders-at-compile-time

Reply:
Removed this incorrect change.